packaging/rpm/microshift.spec

```diff
@@ -729,6 +770,20 @@ fi
 %files ai-model-serving-release-info
 %{_datadir}/microshift/release/release-ai-model-serving-x86_64.json
 
+%files observability
+%dir %{_prefix}/lib/microshift/manifests.d/003-microshift-observability
+%dir %{_sharedstatedir}/microshift-observability
+%config %{_sysconfdir}/microshift/opentelemetry-collector-small.yaml
+%config %{_sysconfdir}/microshift/opentelemetry-collector-medium.yaml
+%config %{_sysconfdir}/microshift/opentelemetry-collector-large.yaml
+%config %{_unitdir}/microshift-observability.service
+%config %{_presetdir}/90-enable-microshift-observability.preset
+%config %{_prefix}/lib/microshift/manifests.d/003-microshift-observability/00-namespace.yaml
+%config %{_prefix}/lib/microshift/manifests.d/003-microshift-observability/01-service-account.yaml
+%config %{_prefix}/lib/microshift/manifests.d/003-microshift-observability/02-cluster-role.yaml
+%config %{_prefix}/lib/microshift/manifests.d/003-microshift-observability/03-cluster-role-binding.yaml
+%config %{_prefix}/lib/microshift/manifests.d/003-microshift-observability/kustomization.yaml
```

Comment:
I'm not sure `%config` is appropriate for these files. I think whatever marked as `%config` does not get removed or overwritten when updating / uninstalling.